knowledge-base/common-change-selected-localization-keys.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Localize selected localization keys with custom strings
```

Comment:
Use title case, see [Capitalization](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).
```suggestion
title: Localize Selected Localization Keys with Custom Strings
```